projects/habitat_ovmm/evaluator.py

```diff
@@ -341,10 +341,14 @@ def remote_evaluate(
 
         import grpc
 
-        from home_robot_hw.utils.eval_ai import evaluation_pb2, evaluation_pb2_grpc
+        try:
+            import evaluation_pb2
+            import evaluation_pb2_grpc
+        except ImportError:
+            from home_robot_hw.utils.eval_ai import evaluation_pb2, evaluation_pb2_grpc
 
         # Wait for the remote environment to be up and running
-        time.sleep(1)
+        time.sleep(60)
```

Comment:
Changing this file back to how it was earlier. The files in `home_robot_hw` are different from the ones in eval_ai repository